tests/unit/authenticators/oauth2-password-grant-test.js

```diff
@@ -117,7 +117,7 @@ describe('OAuth2PasswordGrantAuthenticator', () => {
       authenticator.authenticate('username', 'password');
     });
 
-    it('sends an AJAX request to the token endpoint with client_id Basic Auth header', function(done) {
+    it('(deprecated) sends an AJAX request to the token endpoint with client_id Basic Auth header', function(done) {
```

Comment:
There should also be a test that asserts that the deprecation is triggered at the right point.

Reply:
Tests for deprecation notice now.

Reply:
Sorry - I meant adding a separate test case to test this. We're generally testing one _"thing"_ per test case.